dashboard/src/instrumentation.ts

```diff
@@ -0,0 +1,5 @@
+export const register = async () => {
+  if (process.env.NEXT_RUNTIME === "nodejs") {
+    await import("./instrumentation-node");
+  }
```

Comment:
Not only if runtime, but also we may disable it entirely. Use an environment variable to opt to enable/disable.